_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
+- Metadata
+- Configuration
+- Security
+- Networking
+- Storage
+- Cluster Operation
+- Workloads
+- Architecture
+- API
+- Logging
```

Comment:
Perhaps logging should be moved under a generic "Operations" tag? Otherwise, we would end up with Monitoring, Metrics, Logging, etc.